transportable-udfs-spark_2.11/src/test/scala/com/linkedin/transport/spark/TestSparkTypeFactory.scala

```diff
@@ -64,23 +64,23 @@ class TestSparkTypeFactory {
       "bytesField", "arrField")
     val fieldTypes = Array("varchar", "integer", "bigint", "boolean", "real", "double", "varbinary", "array(integer)")
 
-    val stdStruct = stdFactory.createStruct(fieldNames.toList.asJava, fieldTypes.map(stdFactory.createDataType).toList.asJava)
-    val internalRow = stdStruct.asInstanceOf[PlatformData].getUnderlyingData.asInstanceOf[InternalRow]
+    val row = typeFactory.createStruct(fieldNames.toList.asJava, fieldTypes.map(typeFactory.createDataType).toList.asJava)
```

Comment:
`createStruct` --> `createRowData`

Reply:
That's from the TypeFactory Interface. I am assuming you want to rename 'createStruct' to 'createRowData' for everything.

Reply:
addressed